test/Api.Tests/Utility/TestEmailSender.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace PaderbornUniversity.SILab.Hip.CmsApi.Tests.Utility
 {
-    public class TestEmailSender : IEmailSender
+    public class TestEmailSender : IEmailSender //https://github.com/xunit/xunit/issues/762. It allows only with IDisposable but not with custom service.
```

Comment:
I do not see how the referenced issue has sth to do with our problem ... I think you should just remove this comment